internals/secrethub/run.go

```diff
@@ -38,11 +38,11 @@ var (
 )
 
 // RunCommand runs a program and passes environment variables to it that are
-// defined with --envar or --template flags and secrets.yml files.
+// defined with --var or --template flags and secrets.yml files.
```

Comment:
After this change, how do you distinguish between environment variables and variables to be used in templates?